python/feathub/feature_views/derived_feature_view.py

```diff
@@ -47,6 +47,9 @@ def __init__(
                          {table_name}.{feature_name}, which refers to a feature in the
                          table with the given name, or in the format {feature_name},
                          which refers to a feature in the source table.
+                         ExpressionTransform features that depend on OverWindowTransform
```

Comment:
Would it be simpler to use the following statement in this doc:

For any feature computed by ExpressionTransform or OverWindowTranform, its expression can only depend on the features specified earlier in this list and the features in the source table.

Reply:
Thanks for the suggestion. It is indeed simpler and easier to understand.